src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { t } from '@lingui/macro'
+import { Col, Form, Input, Row, Space } from 'antd'
+import { FormImageUploader } from 'components/inputs/FormImageUploader'
+import { CreateCollapse } from '../../CreateCollapse'
+
+// TODO: Do we want to set validators on these?
+export const ProjectDetailsPage: React.FC = () => {
+  return (
+    <Form name="projectDetails" colon={false} layout="vertical">
+      <Space direction="vertical" size="large" style={{ width: '100%' }}>
+        <Form.Item name="projectName" label={t`Project Name`}>
+          <Input placeholder={t`Enter project name`} />
+        </Form.Item>
+        <Form.Item name="projectDescription" label={t`Project Description`}>
+          <Input.TextArea
+            autoSize={{ minRows: 4, maxRows: 6 }}
+            placeholder={t`Enter project description`}
+          />
+        </Form.Item>
+        <Form.Item name="logo" label={t`Logo`}>
+          <FormImageUploader text={t`Upload`} />
+        </Form.Item>
+        <CreateCollapse>
+          <CreateCollapse.Panel key={0} header={t`Project Links`}>
+            {/* Adding paddingBottom is a bit of a hack, but horizontal gutters not working */}
+            <Row gutter={32} style={{ paddingBottom: '32px' }}>
```

Comment:
**suggestion**: 

```suggestion
            <Row gutter={32} style={{ paddingBottom: '2rem' }}>
```

Reply:
Ah thank you - probably have missed a few of these